kubejs/server_scripts/base/unification/unify_loot.js

```diff
@@ -1,196 +1,80 @@
 // https://github.com/AlmostReliable/lootjs/wiki
 LootJS.modifiers((event) => {
-    // Replace ingots / nuggets / Storage Blocks / Gems in loot chests!
+    // How to use unifyLoot()
+    // tag              -> String - Tag to search for (accepts both tags with and without #)
+    // suffix           -> String - suffix for the EE variant of the item
+    // prefix           -> String - prefix for the EE variant of the item (for example, when you want to unify crushed ores (...:crushed_iron_ore))
+    // ItemStack        -> ItemStack - Original ItemStack that was meant to be dropped
+    // replaceWithRaw   -> Boolean - Should it replace the drop with Raw ore, or return unified version of the item?
+    // count            -> integer - if replaceWithRaw is True, then specifies the amount of raw ore given for each item.
+    function unifyLoot(tag, suffix, ItemStack, prefix, replaceWithRaw, count) {
+        prefix = (typeof prefix == undefined || prefix == null)? "": prefix + "_";
```

Comment:
I think you can just say `!prefix` instead of `(typeof prefix == undefined || prefix == null)`

Reply:
Main idea was to just use `typeof prefix == underfined` and it should work, however for some reason Rhino didn't like that?
But yup, `!prefix` works perfectly 😄